distributed/client.py

```diff
@@ -3546,7 +3604,7 @@ def dump_cluster_state(
         filename: str = "dask-cluster-dump",
         exclude: Collection[str] = (),
         format: Literal["msgpack", "yaml"] = "msgpack",
-    ) -> Awaitable | None:
+    ):
```

Comment:
The previous version will confuse mypy (see mypy guidelines re. `return Any`). I don't think there's an easy way to annotate sync/async client functions short of some major refactoring work.